test/documentation/api-stability.js

```diff
@@ -13,12 +13,28 @@ $(document).ready(function()
         $(this).find('div.memproto').html(new_html);
     }
 
-    var add_labels = function(api_type, entry_type)
+    var add_labels = function(api_type, api_color, entry_obj, entry_type)
     {
         if (entry_type == "member")
+        {
             $(this).find('span.mlabels').prepend('<span class="mlabel ' + api_type + '">' + api_type + '</span>');
+
+            // get header above the item, append the api_type tag to the header and change the bg color
+            prev = entry_obj;
+            do prev = prev.previousSibling; while(prev && prev.nodeType !== 1);
```

Comment:
we seem to have jquery. it should contain an `up` function or something that you could use instead to make this more "robust".

Reply:
I'll check again

Reply:
Thanks @eseiler